src/server/string_family.cc

```diff
@@ -501,30 +503,54 @@ SinkReplyBuilder::MGetResponse OpMGet(bool fetch_mcflag, bool fetch_mcver, const
   return response;
 }
 
-// Either string or future from tiered storage
-struct StringValue {
-  StringValue() : v_{} {
-  }
-  StringValue(std::string s) : v_{std::move(s)} {
-  }
-  StringValue(util::fb2::Future<std::string> f) : v_{std::move(f)} {
+// Helper for building replies for strings
+struct StringReplies {
```

Comment:
Is it only for Gets/Reads? Worth naming accordingly

Reply:
It's used in half of the GET[EX|SET|etc] functions, as well as SET with GET option. StringReplies because it replies with a string. What other name do you suggest?

Reply:
it should be a noun. `GetReplier` my suggestion

Reply:
Have you seen this? :)

Reply:
Will fix it next time 😅